x-pack/plugins/security_solution/public/common/store/data_table/epic_local_storage.ts

```diff
@@ -34,7 +34,7 @@ const {
 
 export const isNotNull = <T>(value: T | null): value is T => value !== null;
 
-const tableActionTypes = [
+const tableActionTypes = new Set([
```

Comment:
The reason I changed all the epic filter action checks to using `Set`s is that they are saving a hundreds of unnecessary iterations. From reload to displaying a timeline, roughly 50 actions are dispatched.

Each epic receives all actions, so just the local-storage epic and the timeline-changed epic alone would in a worst case scenario need 1650 iterations when using `array.includes()`. Using `Set`, it's 100 in a worst case scenario.